woocommerce/woocommerce-android
# [Woo POS][Cash & Receipts] Navigation via navigation component to the cash payment screen

Closes: #13063


### Description

The PR moves mock of the cash payment state to a separate screen. The screen now opened using compose navigation

### Testing information

* More -> POS -> Add product -> Checkout
* Click on take payment button
* Notice a new screen is opened
* Navigate back

### The tests that have been performed

Above

### Images/gif



https://github.com/user-attachments/assets/918bf4f7-badf-4cb1-bdc7-db49608b7c1f




## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
